File: middle/Models/CardModel.cs
namespace PaperDungeon
{
public class CardModel
{
    public string Name { get; set; }
    public List<int> Entity { get; set; } = new List<int>(); 
    public int Target { get; set; }
    public int Aggro { get; set; }
    public int Health { get; set; }
    public int Hot { get; set; }
    public int Shield { get; set; }
    public int Shot { get; set; }
    public int Strengthen { get; set; }
    public int Stot { get; set; }
    public int Reflect { get; set; }
    public int Counter { get; set; }
    public int Interrupt { get; set; }
    public int ExplosivePoisonTrap { get; set; }
    public int ParalyzingTrap { get; set; }

    public CardModel(string name, dynamic properties)
    {
        Name = name;
        Entity = properties.entity ?? new List<int> {0}; 
        Target = properties.target;
        Aggro = properties.aggro;
        Health = properties.health ?? 0;
        Hot = properties.hot ?? 0;
        Shield = properties.shield ?? 0;
        Shot = properties.shot ?? 0;
        Strengthen = properties.strengthen ?? 0;
        Stot = properties.stot ?? 0;
        Reflect = properties.reflect ?? 0;
        Counter = properties.counter ?? 0;
        Interrupt = properties.interrupt ?? 0;
        ExplosivePoisonTrap = properties.explosivePoisonTrap ?? 0;
        ParalyzingTrap = properties.paralyzingTrap ?? 0;
    }
}
}

File: src/engine/models/CardModel.mjs
export class CardModel {
    constructor(name, properties) {
      this.name = name;
      this.entity = properties.entity ? [...properties.entity] : [0];
      this.target = properties.target;
      this.aggro = properties.aggro;
      this.health = properties.health || 0;
      this.hot = properties.hot || 0;
      this.shield = properties.shield || 0;
      this.shot = properties.shot || 0;
      this.strengthen = properties.strengthen || 0;
      this.stot = properties.stot || 0;
      this.reflect = properties.reflect || 0;
      this.counter = properties.counter || 0;
      this.interrupt = properties.interrupt || 0;
      this.explosivePoisonTrap = properties.explosivePoisonTrap || 0;
      this.paralyzingTrap = properties.paralyzingTrap || 0;
    }
  }
